YoutubeExplode/Channels/ChannelClient.cs

```diff
@@ -83,6 +83,6 @@ private Channel Extract(ChannelPageExtractor channelPage)
     {
         // Replace 'UC' in channel ID with 'UU'
         var playlistId = "UU" + channelId.Value.Substring(2);
-        return new PlaylistClient(_http).GetVideosAsync(playlistId, cancellationToken);
+        return new PlaylistClient(_http).GetVideosAsync(playlistId, cancellationToken : cancellationToken);
```

Comment:
Was this change necessary?

```suggestion
        return new PlaylistClient(_http).GetVideosAsync(playlistId, cancellationToken);
```